lib/scrabble.rb

```diff
@@ -1,18 +1,35 @@
 class Scrabble
+  attr_reader :word
+
+  def initialize
+    @word = nil
+  end
 
   def score(word)
-    1
+    @word = word.upcase!
```

Comment:
Looks like we're missing a check to see if the word is nil. Believe `#upcase` will error out if word is `nil`.